distsql/request_builder.go

```diff
@@ -306,6 +306,7 @@ func (builder *RequestBuilder) SetFromSessionVars(sv *variable.SessionVars) *Req
 	}
 	builder.RequestSource.RequestSourceInternal = sv.InRestrictedSQL
 	builder.RequestSource.RequestSourceType = sv.RequestSourceType
+	builder.RequestSource.ExplicitRequestSoureType = sv.ExplicitRequestSoureType
```

Comment:
Noticed some spell error with `Soure` and `reqest`
```suggestion
	builder.RequestSource.ExplicitRequestSourceType = sv.ExplicitRequestSourceType
```